modules/aws/iam/aws-load-balancer-controller.json

```diff
@@ -4,12 +4,24 @@
         {
             "Effect": "Allow",
             "Action": [
-                "iam:CreateServiceLinkedRole",
+                "iam:CreateServiceLinkedRole"
+            ],
+            "Resource": "*",
+            "Condition": {
+                "StringEquals": {
+                    "iam:AWSServiceName": "elasticloadbalancing.amazonaws.com"
+                }
+            }
+        },
+        {
+            "Effect": "Allow",
+            "Action": [
```

Comment:
changes are required due to amazon changes to aws load balancer controller. easiest way to get updated policy was to adjust in our forked repo